File: CJY/BJ/No_1978.js
const rl = require("readline").createInterface({input: process.stdin, output: process.stdout});
let n;

rl.on("line", (s) => {
    if(n) {
        s = s.split(" ");
        let cnt = 0;
        for (let i = 0; i < n; i++) {
            let num = parseInt(s[i]);
            if(num === 1) cnt++;
            for (let j = 2; j < num; j++) {
                if (num % j === 0) {
                    cnt++;
                    break;
                }
            }
        }
        console.log(n - cnt);
        rl.close();
    } else {
        n = parseInt(s);
    }
}).on("close", () => {process.exit();});

File: CJY/BJ/No_1978.java
import java.io.*;

public class No_1978 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        String[] arr = br.readLine().split(" ");
        br.close();

        int cnt = 0;
        for (int i = 0; i < n; i++) {
            int num = Integer.parseInt(arr[i]);
            if(num == 1) cnt++;
            for (int j = 2; j < num; j++) {
                if (num % j == 0) {
                    cnt++;
                    break;
                }
            }
        }

        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        bw.write(String.valueOf(n - cnt));
        bw.flush();
        bw.close();
    }
}
